src/vhost.c

```diff
@@ -389,13 +359,16 @@ struct pg_brick *pg_vhost_new(const char *name, uint64_t flags,
 
 static void vhost_destroy(struct pg_brick *brick, struct pg_error **errp)
 {
-	struct pg_vhost_state *state;
-
-	state = pg_brick_get_state(brick, struct pg_vhost_state);
+	struct pg_vhost_state *state =
+		pg_brick_get_state(brick, struct pg_vhost_state);
 	rte_vhost_driver_unregister(state->socket->path);
 
 	pthread_mutex_lock(&mutex);
-	g_remove(state->socket->path);
+
+	/* If the socket is client, do NOT destroy the existing socket. */
+	if(pg_vhost_is_server(brick) == 0)
```

Comment:
White space between ```if ```and ```(```

Reply:
It's strange that `make style` didn't report it...